### ISSUE TITLE
Error with subliminal from CLI (when doing anything)

### ISSUE THREAD

**Issue by timothymowens** [Diaoul/subliminal#668]
```
$ subliminal download -l en video.s01e01.scene.mkv
Traceback (most recent call last):
  File "/usr/local/bin/subliminal", line 7, in <module>
    from subliminal.cli import subliminal
  File "/usr/local/lib/python2.7/dist-packages/subliminal/cli.py", line 18, in <module>
    from dogpile.core import ReadWriteMutex
ImportError: cannot import name ReadWriteMutex
```

Installed subliminal using pip instructions on Ubuntu 16.04, python v2.7.  Version installed is 2.0.1.

**Comment by Diaoul** [comment]
This is an issue with dogpile.cache new release 0.6.0 from today.

**Comment by Diaoul** [comment]
See https://bitbucket.org/zzzeek/dogpile.cache/issues/99/readwritemutex-is-not-provided-anymore-by

**Comment by timothymowens** [comment]
I guess I picked the wrong day to try out subliminal.  Can I force a specific older verison of dogpile.cache to get subliminal working?

**Comment by timothymowens** [comment]
Actually, I just ran:
`sudo pip install dogpile.cache==0.5.4`
and tested subliminal again and it's working now.

**Comment by Diaoul** [comment]
Fixed with 2.0.2
